e2e/testcases/root_sync_test.go

```diff
@@ -19,7 +19,7 @@ import (
 	"testing"
 	"time"
 
-	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"go.uber.org/multierr"
```

Comment:
We have our own multierror in the status package. Might as well just use that. It's similar.

Reply:
Actually, I'm surprised this didn't change vendor/... Do we already use this uber package elsewhere? Or is it maybe just used by a dependency?

Reply:
The uber package is used in other places. 

Using the multierror in the status package will wrap the error with a KNV999 prefix, which seems to be weird in the e2e test.
```
2 error(s)

[1] KNV9999: fake error 1

For more information, see https://g.co/cloud/acm-errors#knv9999


[2] KNV9999: fake error 2

For more information, see https://g.co/cloud/acm-errors#knv9999
```